### ISSUE TITLE
function-call-argument-newline rule since ESLint 6.2.0

### ISSUE THREAD

**Issue by ybiquitous** [prettier/eslint-config-prettier#102]
The [`function-call-argument-newline`](https://eslint.org/docs/rules/function-call-argument-newline) rule has been added since [ESLint 6.2.0](https://github.com/eslint/eslint/blob/v6.2.0/CHANGELOG.md).

This rule looks to detect [stylistic issues](https://github.com/eslint/eslint/blob/320b7bdb696c0644e838ec649b08c214d27e3bd3/lib/rules/function-call-argument-newline.js#L18), so should eslint-config-prettier disable this rule?

The following is a test case:

```shell
$ eslint -v
v6.2.0

$ prettier -v
1.18.2

$ cat test.js
/*eslint function-call-argument-newline: ["error", "always"]*/

function foo(a, b, c) {
  return a + b + c;
}

foo("one", "two", "three");

$ eslint --no-eslintrc test.js

/tmp/test.js
  7:11  error  There should be a line break after this argument  function-call-argument-newline
  7:18  error  There should be a line break after this argument  function-call-argument-newline

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the `--fix` option.

$ prettier --no-config -c test.js
Checking formatting...
All matched files use Prettier code style!
```

**Comment by lydell** [comment]
Yes, this rule should be disabled. :+1:

Do you want to make a PR?

**Comment by ybiquitous** [comment]
No. I would be glad if you could make a PR instead of me. 😺

**Comment by lydell** [comment]
Released in 6.1.0. Thanks for the report! 👍

**Comment by ybiquitous** [comment]
Thank you!
